src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
For reference code in lets say mup web would look like
<IntlProvider>
  <Form>
    <TextInput placeholder = {intl.formatMessage(trn.placeholder)} />
  </Form>
</IntlProvider>